gradle.properties

```diff
@@ -1,3 +1,6 @@
 org.gradle.jvmargs=-Xmx1536m
 kotlin.coroutines=enable
 kotlin.incremental=true
+marvelPublicKey="212822db30afe0ebc6a83e2343c42708"
```

Comment:
the purpose of leaving keys behind the scenes (home folder `.gradle/gradle.properties` file) is to avoid pushing them to the repo, mainly. You don't need to do this (I guess these are your real api keys)